src/bio/bio_xstream.c

```diff
@@ -1404,6 +1425,24 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id, bool self_polling)
 				DP_RC(rc));
 			goto out;
 		}
+
+		/* After bio_bdevs are initialized, restart SPDK JSON-RPC server if required. */
+		if (nvme_glb.bd_enable_rpc_srv) {
+			if ((!nvme_glb.bd_rpc_srv_addr) || (strlen(nvme_glb.bd_rpc_srv_addr) == 0))
+				nvme_glb.bd_rpc_srv_addr = SPDK_DEFAULT_RPC_ADDR;
+
+			rc = spdk_rpc_initialize(nvme_glb.bd_rpc_srv_addr);
```

Comment:
Can two engines listen on same addr?

Reply:
no, if you had 2 engines and wanted to enable the RPC server on both you would need to specify different socket address for each in the server config file (`sock_addr: /tmp/spdk_0.sock` for example)

Reply:
I see, thanks.